README.md

```diff
@@ -165,23 +168,23 @@ Pandas, Matplotlib, Scipy, PyTorch to name a few.
 If base libraries in this template aren't sufficient you can add more with `pip install library`.
 However, `pip` command installs libraries into your local Python environment. 
 To achieve consistent reproducibility we need to gather information about requirements into project repository. 
-New libraries are added to **`project_requirements.in`** file. When you change this file remember to run:
+New libraries are added to **`project.in`** file. When you change this file remember to run:
```

Comment:
Why `project.in` without `_requirements` but it's present in other .in files?

Reply:
Good question. No valid reason, I'll change this back.